Userland/Services/WindowServer/WindowManager.cpp

```diff
@@ -784,23 +784,33 @@ bool WindowManager::process_ongoing_window_move(MouseEvent& event)
             auto pixels_moved_from_start = event.position().pixels_moved(m_move_origin);
 
             auto event_location_relative_to_screen = event.position().translated(-cursor_screen.rect().location());
-            if (is_resizable && event_location_relative_to_screen.x() <= tiling_deadzone) {
-                if (event_location_relative_to_screen.y() <= tiling_deadzone + desktop_relative_to_screen.top())
+            auto rel_x = event_location_relative_to_screen.x();
+            auto rel_y = event_location_relative_to_screen.y();
+            auto desk_rel_width = desktop_relative_to_screen.width();
+            auto desk_rel_height = desktop_relative_to_screen.height();
```

Comment:
Please do not abbreviate variables unnecessarily. `relative_x`, `desktop_relative_height` etc.

Reply:
Thanks!